src/main.ts

```diff
@@ -3,6 +3,11 @@ import {platformBrowserDynamic} from '@angular/platform-browser-dynamic';
 import {environment} from './environments/environment';
 import {DemoModule} from './app/demo/demo.module';
 
+declare const require;
+
+require('style-loader!../node_modules/intro.js/minified/introjs.min.css');
```

Comment:
On a second thought, I'd try to require it in the directive, to keep all introjs things together